docs/guides/expectations/creating_custom_expectations/how_to_use_custom_expectations.md

```diff
@@ -0,0 +1,208 @@
+---
+title: How to use a Custom Expectation
+---
+import Prerequisites from '../creating_custom_expectations/components/prerequisites.jsx'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Custom [**Expectations**](../../../reference/expectations/expectations.md) are extensions to the core functionality of Great Expectations. Many Custom Expectations may be fit for a very specific purpose, 
+or be at lower levels of stability and feature maturity than the core library. 
+
+As such, they are not available for use from the core library, and require registration and import to become available.
+
+This guide will walk you through the process of utilizing Custom Expectations, whether they were built by you or came from the Great Expectations Experimental Library.
+
+<Prerequisites>
+
+- Created a [Custom Expectation](./overview.md) ***or*** identified a Custom Expectation for use from the [Great Expectations Experimental Library](https://github.com/great-expectations/great_expectations/tree/develop/contrib/experimental/great_expectations_experimental/expectations)
+
+</Prerequisites>
+
+## Steps
+
+<Tabs
+  groupId="expectation-type"
+  defaultValue='custom-expectations'
+  values={[
+  {label: 'Custom Expectations You\'ve Built', value:'custom-expectations'},
+  {label: 'Custom Expectations Contributed To Great Expectations', value:'contrib-expectations'},
+  ]}>
+
+<TabItem value="custom-expectations">
+
+### 1. File placement & import
+
+If you're using a Custom Expectation you've built, 
+you'll need to place it in the `great_expectations/plugins/expectations` folder of your Great Expectations deployment. 
+
+When you instantiate your [**Data Context**](../../../reference/data_context.md), it will automatically make all plugins in the directory available for use, 
```

Comment:
Can we use `TechnicalTag` for these?

Reply:
Totally forgot but absolutely! Super hyped these are around now.